src/components/table-component/table.jsx

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+
+/* 
+ @author [Kurtney J.](https://github.com/xhundo)
+*/
+
+// Table component build with named spaced format
+
+const StyledCell = styled.div`
+  width: 343px;
+  height: 84px;
+  background: rgba(246, 246, 246, 0.1);
+`;
+
+const StyledHeading = styled.p`
+  font-size: 10px;
```

Comment:
where possible, it's better to use the bootstrap className `font-size-x` (replace `x` with whichever size you want) so that way we don't have random usages of pixel-based font sizes in a lot of places

Reply:
I've tried using `fs-6` bootstrap class and this doesn't give me the corresponding look in figma for the headings.

Reply:
Added `$font-size` sass map that includes the entire font sizes (figma layouts mobile/desktop) inside of `custom.scss` sass file.